public/app/percona/dbaas/components/DBCluster/DBClusterLogsModal/PodLogs/PodLogs.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC, useState } from 'react';
+import { Collapse, useStyles } from '@grafana/ui';
+import { PodLogsProps } from './PodLogs.types';
+import { Messages } from './PodLogs.messages';
+import { ContainerLogs } from '../ContainerLogs/ContainerLogs';
+import { getStyles } from './PodLogs.styles';
+
+export const PodLogs: FC<PodLogsProps> = ({ podLogs }) => {
+  const styles = useStyles(getStyles);
+  const { name, isOpen: isPodOpen, containers, events } = podLogs;
+  const [isOpen, setIsOpen] = useState(isPodOpen);
+
```

Comment:
given `isOpen` is based on a prop, shouldn't there be an effect to listen to changes on `isPodOpen` and update `isOpen`?
Otherwise, I think a re-render from above might cause `isPodOpen` to change while `isOpen` is kept the same.

Reply:
It's ok like this, was actually to prevent it to change unless by a user action (which is handled inside the component with the setIsOpen). Is annoying when you are watching logs and they just disappear :D

Reply:
Roger :D